src/Core/Helpers/Argument.Number.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace CodeMonkeys.Core
+{
+    public static partial class Argument
+    {
+        public static void Min(
```

Comment:
Why `Min` ?
`GreaterThan` would be a better name imo

Reply:
I don't think so. For me Min is inclusive. And GreaterThan would be exclusive

Reply:
But you are not checking a minimum here but wether param is greater or equal to value.
The method name should reflect that.

`GreaterOrEqual`

Reply:
Then I will change the implementation here to reflect what I wrote before.

Reply:
This is basically the same but `LessOrEqual`

`Min` is just a name that doesnt say a lot, for me I expect to return the minimum of something (like minimum int, minimum from a collection,...)

Reply:
There is a good issue here about this: https://github.com/safakgur/guard/issues/40